wasm/tests/test.c

```diff
@@ -1790,6 +1790,8 @@ void test_json(void)
 {
     test("json/marshal", opa_value_compare(opa_json_marshal(opa_string_terminated("string")), opa_string_terminated("\"string\"")) == 0);
     test("json/unmarshal", opa_value_compare(opa_json_unmarshal(opa_string_terminated("\"string\"")), opa_string_terminated("string")) == 0);
+    test("json/is_valid_true", opa_value_compare(opa_json_is_valid(opa_string_terminated("\"string\"")), opa_boolean(true)) == 0);
```

Comment:
[nit] We're already dealing with booleans, can we do
```suggestion
    test("json/is_valid_true", opa_cast_boolean(opa_json_is_valid(opa_string_terminated("\"string\"")));
```
and `!opa_cast_boolean(...)` here?

Reply:
Hmm, seems that it does not work the way we would expect for opa_boolean_t. The following fails
`test("json/false_is_false", !opa_cast_boolean(opa_boolean(false)));`

But the following seems to work by referencing the bool field `v` of `opa_boolean_t`,
`test("json/false_is_false", !opa_cast_boolean(opa_boolean(false))->v);`

I am not entirely sure that this is the correct way?